### PR TITLE
Sprint 3

### PR DESCRIPTION
**Pull request by Proka01**
Added cards page.
Added bank accounts page.
Modified permissions pop-up.
Fix log-out -> log-in founded bug.

In bank-account-service and card-service REPLACE !!! MOCKED functions with actual functions !!!!!
Actual functions are written and ready to be used when backend is done during PR merging.

### CODE REVIEW

File: src/app/app.routes.ts [hunk 1]
```diff
@@ -39,6 +41,9 @@ export const routes: Routes = [
      canActivate: [employeeGuard]
   },
   {path: 'forex/:ticker', component: ForexViewComponent},
+  //TODO: Add customer guard
+  { path: 'bank-accounts', component: BankAccountsComponent},
+  { path: 'cards', component: CardsComponent}
```
Comment (RadenkovicZarko) [inline src/app/app.routes.ts:46]: fali guard

File: src/app/app.routes.ts [hunk 2]
```diff
@@ -39,6 +41,9 @@ export const routes: Routes = [
      canActivate: [employeeGuard]
   },
   {path: 'forex/:ticker', component: ForexViewComponent},
+  //TODO: Add customer guard
+  { path: 'bank-accounts', component: BankAccountsComponent},
```
Comment (RadenkovicZarko) [inline src/app/app.routes.ts:45]: fali guard

**Review by RadenkovicZarko — CHANGES_REQUESTED** [review]
Sve super, dodaj guard-ove samo na rute :)

File: src/app/app.routes.ts (hunk 1 above)
Comment (Proka01) [reply src/app/app.routes.ts]: Guard dodat i testiran.

File: src/app/app.routes.ts (hunk 2 above)
Comment (Proka01) [reply src/app/app.routes.ts]: Guard dodat i testiran.

**Comment by Proka01** [comment]
Guard dodat i testiran. 
All done.

**Pull request merged**